cmd/serverless-init/log/log.go

```diff
@@ -81,3 +96,9 @@ func addFileTailing(logsAgent logsAgent.ServerlessLogsAgent, source string, tags
 func isEnabled(envValue string) bool {
 	return strings.ToLower(envValue) == "true"
 }
+
+// enabled by default
```

Comment:
It looks like the behavior was changed to disabled by default but this comment still says enabled by default